conda/meta.yaml

```diff
@@ -1,19 +1,12 @@
 package:
   name: conda
-  version: 3.8.4
+  version:  '{{ environ["GIT_DESCRIBE_TAG"] }}.post{{ environ["GIT_DESCRIBE_NUMBER"] }}+{{ environ["GIT_DESCRIBE_HASH"] }}'
 
 source:
-  git_url: git@github.com:conda/conda.git
-  git_tag: 3.8.4
+  git_url: https://github.com/conda/conda.git
 
 build:
-  # These are present when the new environment is created
-  # so we have to exempt them from the list of initial files
-  # for conda-build to realize they should be included.
-  always_include_files:
-    - bin/activate
-    - bin/deactivate
-    - bin/conda
```

Comment:
(When I installed the built package, I did get the version of conda that I expected- with the fix for https://github.com/conda/conda/pull/1766 - would have been v obvious if it wasn't there as I then tested that I could install all the cases that I previously couldn't)